app/worker/smtp.go

```diff
@@ -15,6 +15,8 @@
 package worker
 
 import (
+	"github.com/pkg/errors"
```

Comment:
I can be wrong, but shouldn't you move this import to the next block, because it is not a standard lib?
I know it is just a matter of style, but I believe this is how we do it in other packages.

Reply:
> I can be wrong, but shouldn't you move this import to the next block, because it is not a standard lib?
> I know it is just a matter of style, but I believe this is how we do it in other packages.

[sure!](https://github.com/mendersoftware/workflows/pull/24/commits/5765e1a4d2b4481692d162787b8a5326a6663178#diff-71d3b7a06a584ee746ac646a87913b58R25)